Makefile

```diff
@@ -104,6 +104,10 @@ ifeq ($(WITH_K8S), true)
   EXTRA_ARGS+=-analyzer.topology.probes=k8s
 endif
 
+ifeq ($(WITH_HELM), true)
```

Comment:
why we should have this optional build ? (should be enable all the time)

Reply:
if we have a `WITH_K8S` should we not also have a `WITH_HELM` ?

Reply:
we use flags to build a specific feature in skydive (like dpdk, prof, ...)
In case of k8s,helm, there are probes, it's not compilation/build related